azurerm/resource_arm_application_gateway.go

```diff
@@ -2158,7 +2409,17 @@ func flattenApplicationGatewayURLPathMaps(input *[]network.ApplicationGatewayURL
 						ruleOutput["name"] = *rule.Name
 					}
 
+					// TODO: Note below: ruleProps versus props - bug?
 					if ruleProps := rule.ApplicationGatewayPathRulePropertiesFormat; props != nil {
```

Comment:
agreed, this looks like a bug; would you mind fixing this? :)

Reply:
Done